packages/host/app/components/matrix/room.gts

```diff
@@ -586,16 +585,13 @@ export default class Room extends Component<Signature> {
     return this.autoAttachmentResource.cards;
   }
 
-  // using enqueue task on this ensures that updates don't step on each other
-  private updateSkillIsActiveTask = enqueueTask(
```

Comment:
I’m sure it didn’t since you’re making this change, but why didn’t `enqueueTask` work?

Reply:
It did work, but it doing in the component means it would be bypassed by commands that enable/disable skills